### PR TITLE
Attempt to fix mason_client fetch

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: prow/mason_lib.sh [hunk 1]
```diff
@@ -24,9 +24,8 @@ function mason_cleanup() {
 }
 
 function get_resource() {
-  go get istio.io/test-infra/boskos/cmd/mason_client
+  (cd /tmp; go get istio.io/test-infra/boskos/cmd/mason_client)
```
Comment (rlenglet) [inline prow/mason_lib.sh]:
```suggestion
  (cd /tmp && go get istio.io/test-infra/boskos/cmd/mason_client)
```
??

Comment (rlenglet) [reply prow/mason_lib.sh]: Ha. The linter found the same issue. :)

**Comment by howardjohn** [comment]
Ahhhhh this isn't enough. the image here is using an old go version which fails the `go get` command. Maybe can do it with GO111MODULEs

**Pull request merged**